specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -1160,6 +1160,28 @@ model AcsMessageReceivedEventData extends AcsMessageEventData {
   interactive: AcsMessageInteractiveContent;
 }
 
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.AdvancedMessageAnalysisCompleted event. */
+interface AcsMessageAnalysisCompletedEventData extends AcsMessageEventData {
```

Comment:
The following lines need to be added to the client.tsp file also for successful generation

```
@@usage(Microsoft.EventGrid.SystemEvents.AcsMessageAnalysisCompletedEventData,
  Usage.output
);
@@access(Microsoft.EventGrid.SystemEvents.AcsMessageAnalysisCompletedEventData,
  Access.public
);
```

@l0lawrence can provide further clarification